packages/react/src/tags/Tags.stories.tsx

```diff
@@ -58,9 +55,11 @@ export const Modular = () => (
 			Tags:
 		</Text>
 		<TagsList>
-			<Tag href="#">Foo</Tag>
-			<Tag href="#">Bar</Tag>
-			<Tag href="#">Baz</Tag>
+			{exampleLinks.map(({ href, label }, idx) => (
```

Comment:
This was causing an accessibility warning in Storybook, which has now been fixed.